src/typings/electron-ipc.d.ts

```diff
@@ -90,11 +91,12 @@ interface IpcRendererEvents {
   'service-changed': (services: import('@/k8s-engine/k8s').ServiceEntry[]) => void;
 
   // #region Images
-  'kim-process-cancelled': () => void;
-  'kim-process-ended': (exitCode: number) => void;
-  'kim-process-output': (data: string, isStdErr: boolean) => void;
+  'image-process-cancelled': () => void;
+  'image-process-ended': (exitCode: number) => void;
+  'image-process-output': (data: string, isStdErr: boolean) => void;
```

Comment:
```suggestion
  'images-process-cancelled': () => void;
  'images-process-ended': (exitCode: number) => void;
  'images-process-output': (data: string, isStdErr: boolean) => void;
```
To make it match the existing `images-` events.

(I think we should make it into categories, as in `images/process-cancelled`, but that can be a future thing.)